src/particleSim/Particle.cpp

```diff
@@ -36,10 +38,13 @@ Particle::Particle(std::array<double, 3> x_arg, std::array<double, 3> v_arg,
   type = type_arg;
   f = {0., 0., 0.};
   old_f = {0., 0., 0.};
-  std::cout << "Particle generated!" << std::endl;
+  Logger::getInstance().trace("Particle generated!");
 }
 
-Particle::~Particle() { std::cout << "Particle destructed!" << std::endl; }
+Particle::~Particle() {
+  // std::cout << "Particle destructed!" << std::endl;
+  // Logger::getInstance().info("Particle destructed!");
```

Comment:
"Dead" code should be removed. Is there a reason why the log message was commented out.